pkg/storage/db.go

```diff
@@ -689,22 +688,33 @@ func (db *DB) insertNetworkChangeEvent(ctx context.Context, timestamp time.Time,
 
 // FetchAll gets all the assets present at the specified time
 func (db *DB) FetchAll(ctx context.Context, when time.Time, count uint, offset uint, typeFilter string) ([]domain.CloudAssetDetails, error) {
-	return db.runQuery(ctx, bulkResourcesQuery, when, typeFilter, count, offset)
+	return db.runQuery(ctx, 1, bulkResourcesQuery, when, typeFilter, count, offset)
```

Comment:
is there a reason the version is hardcoded to 1?

Reply:
This is because it queries the old database, which is version 1. I think we need to update this function in a new PR. Thank you for asking.

Reply:
ReadsFromNewSchemaVersion from https://github.com/asecurityteam/asset-inventory-api/pull/42/files is addressing this issue.

Reply:
I will fix this PR to use ReadNewSchemaVersion.